packages/core/src/Modules/Cashier/Containers/cashier.jsx

```diff
@@ -1,19 +1,18 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import { withRouter } from 'react-router-dom';
 import {
```

Comment:
Why is this file edited? `eslint/prettier`?

Reply:
It was just a test, but now, all changes have been discarded.